build/nlu-regression.js

```diff
@@ -0,0 +1,218 @@
+var http = require('http')
+var fs = require('fs')
+const { exit } = require('process')
+
+const repoRootDir = `${__dirname}/..`
+const nluTestingDir = `${repoRootDir}/modules/nlu-testing/`
+
+async function post(path, content, headers) {
+  var post_options = {
+    host: '127.0.0.1',
+    port: '3000',
+    path,
+    method: 'POST',
+    headers,
+    timeout: 2000
+  }
+
+  return new Promise(function(resolve, reject) {
+    var post_req = http.request(post_options, function(res) {
+      res.setEncoding('utf8')
+
+      let receivedData = ''
+      res.on('data', function(chunk) {
+        receivedData += chunk
+      })
+      res.on('error', function(err) {
+        reject(err)
+      })
+      res.on('end', function() {
+        resolve(receivedData)
+      })
+    })
+
+    // post the data
+    post_req.write(content)
+    post_req.end()
+  })
```

Comment:
Why not simply use axios like everywhere else ? You'd save like 30 lines just in this method.